src/components/sections/SectionHeader.tsx

```diff
@@ -1,15 +1,18 @@
 import { AppliedFiltersCssClasses } from '../AppliedFilters';
 import { AppliedFiltersDisplay } from '../AppliedFiltersDisplay';
-import { useComposedCssClasses, CompositionMethod } from '../../hooks/useComposedCssClasses';
 import { CollectionIcon } from '../../icons/CollectionIcon';
 import { AppliedQueryFilter, SelectableFilter as DisplayableFilter, useAnswersState } from '@yext/answers-headless-react';
 import classNames from 'classnames';
 import { useAnalytics } from '../../hooks/useAnalytics';
 import { VerticalLink } from '../../models/verticalLink';
 import { useCallback } from 'react';
 
+const DEFAULT_CSS_CLASSES: SectionHeaderCssClasses = {};
```

Comment:
What do you think of "CSS_CLASSES_FALLBACK"?  since 'default' is similar in meaning to 'builtIn'